src/index.ts

```diff
@@ -8,7 +8,7 @@ async function run(): Promise<void> {
 
     // Get inputs
     // https://docs.github.com/en/rest/releases/releases#create-a-release
-    const tagName = core.getInput('tag_name', { required: false }) || github.context.ref.replace('refs/tags/', '')
+    const tagName = github.context.ref.replace('refs/tags/', '')
```

Comment:
The `tag_name` is an optional inputs with an empty string as defaults .

In a case, you can specify a `tagName` you like rather than from context.